vpr/src/base/read_options.cpp

```diff
@@ -1324,6 +1324,11 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .action(argparse::Action::STORE_TRUE)
         .default_value("off");
 
+    stage_grp.add_argument<bool, ParseOnOff>(args.do_analytical_placement, "--analytical_place")
+        .help("Run analytical placement")
```

Comment:
I'd expand on this .help.  Should say analytical placement uses an integrated packing & placement algorithm.

(Reason to explain this: otherwise it would make more sense for us to just have another set of placement options, but not a different stage).